package.json

```diff
@@ -54,7 +54,7 @@
     "okapiInterfaces": {
       "alternative-title-types": "1.0",
       "call-number-types": "1.0",
-      "browse": "0.6 1.0",
+      "browse": "0.6 1.0 1.5",
```

Comment:
Let's add this to `optionalOkapiInterfaces` instead. This way it won't mean a breaking change